plaidml2/bridge/keras/backend_test.py

```diff
@@ -476,21 +477,39 @@ def testFlatten(self, b, x):
     def testEye(self, b, *args):
         return b.eye(*args)
 
-    @unittest.skip('TODO: convert to EDSL')
     def testTileIdentity(self):
-        x = pkb.variable(m(3))
-        op = tile.Operation('function (I[N]) -> (O) { O = I; }', [('I', x)],
-                            [('O', tile.Shape(x.shape.dtype, (3,)))])
-        output = op.sole_output().eval()
+        I = pkb.variable(m(3)).tensor
+        N = edsl.TensorDim()
+        I.bind_dims(N)
+        O = edsl.TensorOutput(N)
+        i = edsl.TensorIndex()
+        O[i] = I[i]
+        opId = pkb._KerasNode('tile_identity', name='opId', tensor=O)
```

Comment:
Actually, I think the equivalent expression would be to pass `I` into `tensor=` kwarg of `_KerasNode`. Basically we're testing that inputs can also be outputs. I'm not actually sure if this one will pass right now :)

Reply:
tried passing I into _KerasNode, see updated code. The test breaks with I[i]=I[I]. Not sure how to write this one.

Reply:
I re-read this and now understand what you were intending; we'll include this change in the PR.